src/pint/scripts/event_optimize_multiple.py

```diff
@@ -339,13 +331,13 @@ def main(argv=None):
         if tname[-6:] == "pickle" or tname == "analytic":
             # Analytic template
             try:
-                gtemplate = cPickle.load(file(tname))
-            except:
+                gtemplate = pickle.load(file(tname))
```

Comment:
Why are you switching from `cPickle` to `pickle`?

Reply:
cPickle was discontinued in python 3. Since we are no longer supporting python2, this part will not work.
https://stackoverflow.com/questions/49579282/cant-find-module-cpickle-using-python-3-5-and-anaconda

Reply:
Quote from StackOverflow:
>There is no cPickle in Python 3. Just import pickle. pickle will automatically use the C accelerator.